course_info/tests/jasmine/controllers/DetailsControllerSpec.js

```diff
@@ -13,6 +13,14 @@ describe('Unit testing DetailsController', function () {
         '=api%2Fcourse%2Fv2%2Fcourse_instances%2F' + courseInstanceId +
         '%2Fpeople%2F&-source=xreg_map';
 
+    var deleteSiteUrl =
+        '/angular/reverse/?djng_url_name=icommons_rest_api_proxy&djng_url_args' +
+        '=api%2Fcourse%2Fv2%2Fcourse_instances%2F' + courseInstanceId + '%2Fsites%2F999%2F';
```

Comment:
This hardcodes the site ID -- in other tests we're adding it as part of the test. Just something to be aware of, that the other URLs here are generic/'base'-type URLs.